### PR TITLE
Fixing `includes` to properly build and set the association.

### PR DESCRIPTION
**Pull request by asedge**
Prior to this fix `ActiveForce::ActiveQuery` would properly fetch the associated records but would not know how to assign them so referencing the association would result in another query to Salesforce.  We added a mapping of Salesforce relation names to the defined association name to accomplish this.

### CODE REVIEW

File: lib/active_force/association/has_many_association.rb [hunk 1]
```diff
@@ -1,6 +1,10 @@
 module ActiveForce
   module Association
     class HasManyAssociation < Association
+      def sfdc_association_field
+        relationship_name.gsub /__c\z/, 's__r'
```
[the remark this answers is not in the document]
Comment (JeffLuckett) [reply lib/active_force/association/has_many_association.rb:5]: Definitely want to find a more robust pluralizer.

**Review by JeffLuckett — COMMENTED** [review]
Looks good to me ... do you need to bump the patch version?

**Comment by asedge** [comment]
> Looks good to me ... do you need to bump the patch version?

Yeah, good call.  I'll also update the `CHANGELOG`.

File: lib/active_force/association/belongs_to_association.rb [hunk 2]
```diff
@@ -1,8 +1,17 @@
 module ActiveForce
   module Association
     class BelongsToAssociation < Association
+
+      def relationship_name
+        options[:relationship_name] || default_relationship_name
```
Comment (JeffLuckett) [inline lib/active_force/association/belongs_to_association.rb:6]: 👍

**Review by JeffLuckett — COMMENTED** [review]
:shipit:

**Review by JeffLuckett — APPROVED** [review]

**Pull request merged**